e2e-tests/insurance/cypress/e2e/journeys/policy/broker-zero-addresses/broker-zero-addresses-page.spec.js

```diff
@@ -0,0 +1,79 @@
+import { headingCaption } from '../../../../../../partials';
+import { brokerZeroAddressesPage } from '../../../../../../pages/insurance/policy';
+import { body } from '../../../../../../pages/shared';
+import { PAGES } from '../../../../../../content-strings';
+import { INSURANCE_ROUTES } from '../../../../../../constants/routes/insurance';
+
+const CONTENT_STRINGS = PAGES.INSURANCE.POLICY.BROKER_ZERO_ADDRESSES;
+const {
+  ROOT,
+  POLICY: { BROKER_ZERO_ADDRESSES_ROOT },
+} = INSURANCE_ROUTES;
+
+const { outro } = brokerZeroAddressesPage;
+
+const mockPostcode = 'W1A 1AA';
+
+const baseUrl = Cypress.config('baseUrl');
+
+context('Insurance - Policy - Broker - zero addresses page - As an exporter ... TODO', () => {
```

Comment:
could we add a TODO and ticket number comment as well please?

Reply:
Updated, we can just include it in the description